app/scripts/modules/core/delivery/triggers/NextRunTag.tsx

```diff
@@ -9,13 +10,14 @@ import { timestamp } from 'core/utils/timeFormatters';
 
 interface IProps {
   pipeline: IPipeline;
-};
+}
 
 interface IState {
   nextScheduled: number;
   hasNextScheduled: boolean;
-};
+}
```

Comment:
Haha, the irony of the tslint semicolon rule... enforce semicolons on interface, or ignore linting semicolons on interfaces altogether.